Makefile

```diff
@@ -75,6 +75,16 @@ ps2_drivers:
 	$(MAKE) -C $@ install
 	$(MAKE) -C $@ clean
 
+ps2stuff:
+	git clone --depth 1 https://github.com/ps2dev/ps2stuff
+	$(MAKE) -C $@ install
+	$(MAKE) -C $@ clean
+
+ps2gl:
```

Comment:
if I'm not mistaken, `ps2gl` depends on `ps2stuff` so you need to add that dependency here:
```make
ps2gl: ps2stuff
    git clone --depth 1 https://github.com/ps2dev/ps2gl
    ....
```